### DISCUSSION TITLE
Code Search and Navigation Check-in

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#51193] [Other Feature Feedback, Questions, & Ideas]
👋 Hello everyone! Hope you’re having a great month so far. 

GitHub Code Search and Code View are [available in public beta](https://github.blog/changelog/2023-02-23-no-more-waitlist-code-search-and-code-view-are-available-to-all-in-public-beta/) for everyone! With Code Search, you can understand your code better as it brings relevant results with incredible speed and with Code View you can dig deeper into billions of lines of public code with an integrated browsing experience and code navigation. There are a ton of new features to try out, so start [exploring](https://docs.github.com/en/search-github/github-code-search/about-github-code-search) at your own pace. Head over to this [discussion](https://github.com/orgs/community/discussions/48301) to share your feedback and ask any questions.

## Changes to the code search API 🚨

We are preparing to bring powerful [new code search](https://github.com/features/code-search) capabilities to GitHub. As part of that effort, on April 10, 2023, we will make several changes to the code search API. Please[ read this changelog post](https://github.blog/changelog/2023-03-10-changes-to-the-code-search-api/) for more details.

To prepare for these changes, make sure your code [handles rate limiting](https://docs.github.com/en/rest/search?apiVersion=2022-11-28#rate-limit). And if you’re using code search to track changes or find security vulnerabilities in your codebase, consider [using webhooks](https://docs.github.com/en/rest/webhooks?apiVersion=2022-11-28#repository-webhooks) or [GitHub Advanced Security](https://docs.github.com/en/rest/webhooks?apiVersion=2022-11-28#repository-webhooks).

## We need your help - Want to flex some solving muscles? 🧠

We are a strong community helping each other by answering queries related to GitHub Code Search and Navigation. I saw these questions haven’t got any response yet and would love to see you contributing and helping. 

- https://github.com/orgs/community/discussions/50861 
- https://github.com/orgs/community/discussions/50171 
- https://github.com/orgs/community/discussions/49810 
- https://github.com/orgs/community/discussions/48151 

## Help us stay categorized! 🤝

The Code Search and Navigation category is for asking your questions and sharing feedback about the product Code Search and Code Navigation. Please use the appropriate category, or if one doesn’t fit, use the General category. This way your post will most likely get seen and responded to by an expert.

Also, we have updated our “New Discussion” category form to help you post in the right category. 
- If you have a question, please select the “Question” topic area. 
- If you’ve got feedback to share, select the “Feedback” topic area. 
- Before posting your discussion, choose the “Code Search and Navigation” label on the right.

![Screenshot 2023-03-26 at 1 05 44 PM](https://user-images.githubusercontent.com/95834130/228143498-70f90bb1-bd59-4bf0-8f7b-4ea9e5553f9b.png)

Thank You! for being such amazing community members. With that, it's time to wrap up this check-in. I’ll be around next month with another one! ⏭️

**Comment by svennergr** [comment]
The deprecation period for those API changes + the bugs we were seeing in API responses over the last days where miserable.

Please bring back `sort:indexed`: https://github.com/orgs/community/discussions/52932

**Reply by hifiveszu** [reply]
can't agree more.
